contributing/add-examples-to-documentation.md

```diff
@@ -0,0 +1,41 @@
+# Adding Examples to Generated Documentation
+
+The Cloud Control provider is 100% generated from the AWS CloudFormation Schema *including* the documentation. This means that at the resource/attribute level any changes must be made in the CloudFormation Schema and cannot be modified within the provider itself. However examples can be added to the generated documentation in order to provide working examples or best practices when using the provider.
+
+The Terraform AWS Cloud Control provider uses [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs) to generate the documentation found in the [Terraform Registry](https://registry.terraform.io/providers/hashicorp/awscc/latest/docs). Full details of the tool can be found on its GitHub repository. In summary, the tool uses the resource schema to generate and populate Terraform Registry compatible documentation.
```

Comment:
Consistency between _Terraform AWS Cloud Control provider_ and _Cloud Control provider_.